edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/Startup.cs

```diff
@@ -221,5 +227,62 @@ string GetStoragePath()
             var value = this.Configuration.GetValue<T>(key);
             return EqualityComparer<T>.Default.Equals(value, default(T)) ? Option.None<T>() : Option.Some(value);
         }
+
+        Option<IMetricsRoot> BuildMetricsCollector()
+        {
+            Option<IMetricsRoot> metricsCollector = Option.None<IMetricsRoot>();
+            bool collectMetrics = this.Configuration.GetValue("CollectMetrics", false);
+            if (!collectMetrics)
+            {
+                return metricsCollector;
+            }
+
+            string metricsStoreType = this.Configuration.GetValue<string>("MetricsStoreType");
+            
+            if (metricsStoreType == "influxdb")
+            {
```

Comment:
Looks like there are a number of config settings related to this.. can we instead create a section for metrics related config? Might be cleaner that way.